<?xml version="1.0"?>

<launch>
    <!-- Robot namespace -->
    <arg name="robot_name" default="my_gen3"/>
    <arg name="pose_input" default="/pcl_closest_obj/output_pose"/>
    <arg name="start_delay_seconds" default="0"/>

    <!-- Launch the example in that namespace -->
    <!-- node name="full_arm_movement_python" pkg="robothon2023" type="full_arm_movement.py" output="screen" launch-prefix="bash -c 'sleep $(arg start_delay_seconds); $0 $@' ">
        <param name="robot_name" value="$(arg robot_name)"/>
    </node -->

    <node pkg="robothon2023" type="pick_and_place.py" name="pick_and_place" output="screen">
        <rosparam command="load" file="$(find robothon2023)/config/joint_angles.yaml"/>
        <rosparam command="load" file="$(find robothon2023)/config/boundary_safety.yaml"/>
        <rosparam command="load" file="$(find robothon2023)/config/task_params.yaml"/>
        <rosparam command="load" file="$(find robothon2023)/config/trajectories.yaml"/>
        <remap from="~pose_in" to="$(arg pose_input)" />
    </node>
</launch>
